### DISCUSSION TITLE
2022-12-09-talk

### DISCUSSION THREAD

**Discussion by giscus** [tw93/tw93.github.io#127] [General]
# 工程师如何明白的做事情 - Tw93

Hi，我是 Tw93，是一个前端工程师，会一点 Swift / Rust / Python，喜欢折腾各种新技术。Share最近给团队同学的一个分享，作为工程师而言如何明白的做事情。

https://tw93.fun/2022-12-09/talk.html

**Comment by liudanking** [comment]
言简意赅，本slide本身也是把事情讲明白的范本。

**Comment by cwandev** [comment]
有启发！

**Comment by jonymei** [comment]
应该用「工程师如何明白地做事情」，「做」是动词，前面的副词用「地」。

**Reply by tw93** [reply]
谢谢指出，下次我注意这个

**Comment by UCASer-Fan** [comment]
获益匪浅，希望能在工作中用起来

**Comment by byte92** [comment]
高质量分享 希望可以学以致用

**Comment by XYZ-qiyh** [comment]
第4点是什么意思呢？
“有完成事情的PM能力” PM是指项目经理/产品经理？

**Reply by tw93** [reply]
可以理解 假如一个项目安排在你手里负责，你能够确保如期高质量从头到尾完成。这个过程肯定需要协调人，商量，讨论，工期制定的，其实有不少学问和技巧。

**Reply by isbasex** [reply]
博主有什么可分享的吗 关于pm相关的技巧，尤其是在大公司

**Reply by tw93** [reply]
其实很简单，就是把公司的事情当做自己的事情去做，认真负责，进度跟到底，有问题及时反馈以及寻求帮助，然后工程师需要有技术判断力，有这些基本上够了
